core/src/View/Kokkos_BasicView.hpp

```diff
@@ -619,6 +629,11 @@ class BasicView {
   template <class, class, class, class>
   friend class BasicView;
 };
+
+#if defined(KOKKOS_ENABLE_HPX) && !defined(__clang__)
+#pragma GCC diagnostic pop
```

Comment:
doesn't this need to happen independent of clang? The diagnostic push up there is just dependent on HPX?

Reply:
Fixed.